apps/ledger-live-mobile/src/screens/AddAccounts/01-SelectCrypto.tsx

```diff
@@ -57,6 +57,7 @@ export default function AddAccountsSelectCrypto({ navigation, route }: Props) {
   const cronos = useFeature("currencyCronosMobile");
   const songbird = useFeature("currencySongbirdMobile");
   const flare = useFeature("currencyFlareMobile");
+  const near = useFeature("currencyNearMobile");
```

Comment:
With conditional feature flags we don't need a seperate flag for desktop and mobile, you can use the same flag `currencyNear`